decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -40,7 +40,7 @@
     settings.attribute :default_sort_order, type: :select, default: "default", choices: -> { POSSIBLE_SORT_ORDERS }
     settings.attribute :official_proposals_enabled, type: :boolean, default: true
     settings.attribute :comments_enabled, type: :boolean, default: true
-    settings.attribute :comments_max_length, type: :integer, required: true
+    settings.attribute :comments_max_length, type: :integer, default: 1000, required: true
```

Comment:
Required `true` is fine to make it consistent with others but I would leave the `default` out because otherwise we are forcing the value on components (rather than using the default).
```suggestion
    settings.attribute :comments_max_length, type: :integer, required: true
```